src/Chapter1.hs

```diff
@@ -417,8 +418,10 @@ task is to specify the type of this function.
 49
 -}
 
+squareSum :: Int -> Int -> Int
 squareSum x y = (x + y) * (x + y)
 
+--squareSum :: Num a => a -> a -> a type
```

Comment:
Yes, you can do so too! We will explain what it exactly means in Haskell later, but for now, you can think of it as a `Num`eric object.